config/routes.rb

```
@@ -1,9 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  devise_for :users
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  namespace :api do
+    devise_for :users, module: 'api/users', defaults: { format: :json }
+  end
```

because we overrode the controllers ourselves, let's specify which action points to which controller explicetely. For this, we should use:
```ruby
devise_for :users, defaults: { format: :json }, controllers: {
      sessions: 'api/users/sessions',
      registrations: 'api/users/registrations',
      passwords: 'api/users/passwords'
    }
```
this way is easier for the next developer to see which actions are the default devise controllers, and which are custom.

Also you can skip all the other controllers (except passwords, registrations and sessions) for now, because they are not used (yet!).

We can address the ones we'll use in a sepparate PR, in order to keep this one slimmer.

---

Should I just leave the password, registration and session controller in this PR then?